.yamato/config/__shared.metafile

```diff
@@ -350,7 +350,7 @@ platforms:
           - command: curl -s https://artifactory.prd.it.unity3d.com/artifactory/unity-tools-local/utr-standalone/utr --output utr
             retries: 2
           - chmod +x utr
-          - command: scp -i ~/.ssh/id_rsa_macmini -r $YAMATO_SOURCE_DIR bokken@$BOKKEN_DEVICE_IP:~/Graphics
+          - command: rsync -uaqP -e 'ssh -i ~/.ssh/id_rsa_macmini' --exclude=".*" --inplace $YAMATO_SOURCE_DIR bokken@$BOKKEN_DEVICE_IP:~/
```

Comment:
The arguments look good to me, looking at the man.
Except I'm wondering why the `exclude=".*"`?

Reply:
Closed PR, the motivation was that to skip `.` files/folders (e.g. `.git/` as well as others like `.yamato/`).